src/Core/Manifest/Module.php

```diff
@@ -301,7 +303,7 @@ public function getResourcePath($path)
     }
 
     /**
-     * @deprecated 4.0.0:5.0.0 Use ->getResource($path)->getURL() instead
+     * @deprecated 4.0.0 Use ->getResource($path)->getURL() instead
```

Comment:
```suggestion
     * @deprecated 4.0.0 Use getResource($path)->getURL() instead
```
Need to update the version in the notice below

Reply:
Updated